README.md

```diff
@@ -1,11 +1,15 @@
-# dave
+# Description
+Dave automates virtually everything is a general-purpose Discord bot designed to help server operators administrate and engage with their community.
```

Comment:
This is false. Dave does not automate anything in its current state. (Also grammar)

Reply:
I would prefer dave to be kept lowercase as part of the branding

Reply:
> I would prefer dave to be kept lowercase as part of the branding

I modified the readme in the commit to make all reference to dave lowercase and removed fluff text that did not add much value.